README.md

```diff
@@ -1105,6 +1110,59 @@ However, the following rules of thumb are useful to keep in mind:
   Alternatively, the entire OrientDB class and all edges that may point to it may be omitted
   entirely from the GraphQL schema.
 
+## Query cost estimation
+
+Due to GraphQL's easy-to-use nature, queries that use a lot of memory and/or have a long execution
```

Comment:
Elaborate on why it's easy to write queries that take up a lot of space and time, i.e. bc we have arbitrary levels of nesting

Reply:
I removed this part.